src/libs/actions/Policy/Member.ts

```diff
@@ -326,7 +335,16 @@ function removeMembers(accountIDs: number[], policyID: string) {
         ReportUtils.buildOptimisticClosedReportAction(sessionEmail, policy?.name ?? '', CONST.REPORT.ARCHIVE_REASON.REMOVED_FROM_POLICY),
     );
 
-    const announceRoomMembers = removeOptimisticAnnounceRoomMembers(policy?.id, policy?.name ?? '', accountIDs);
+    const announceRoomMembers = removeOptimisticRoomMembers(CONST.REPORT.CHAT_TYPE.POLICY_ANNOUNCE, policy?.id, policy?.name ?? '', accountIDs);
+    const adminRoomMembers = removeOptimisticRoomMembers(
+        CONST.REPORT.CHAT_TYPE.POLICY_ADMINS,
+        policy?.id,
+        policy?.name ?? '',
+        accountIDs.filter((accountID) => {
+            const login = allPersonalDetails?.[accountID]?.login;
+            return login && policy?.employeeList?.[login]?.role !== CONST.POLICY.ROLE.USER;
+        }),
+    );
```

Comment:
Are `AUDITOR`s members of the `#admins` room too? Asking because I don't see them mentioned here https://github.com/Expensify/App?tab=readme-ov-file#workspace-admin-room

Reply:
Asked in slack https://expensify.slack.com/archives/C01GTK53T8Q/p1739640700642219

Reply:
The BE adds auditor as the admins member, so I guess yes.

Reply:
I think we should compare with `role === ADMIN || role === AUDITOR` to account for further changes in case another role is added

Reply:
Fair point. Updated.